### ISSUE TITLE
Deny usage of special characters in text fields

### ISSUE THREAD

**Issue by vohmar** [internetee/auction_center#486]
There is currently no limitations on user and profile forms for the characters that are allowed to be used. So we few good examples of user data in the auction system already

```
given_names                      | {{7*7}}
surname                          | {7*7}23x23${7*7}
```

```
name                   | "><svg/onload=confirm(1)>
vat_code               | '-sleep(10)#
street                 | '-sleep(10)#
city                   | '-sleep(10)#
state                  | 
postal_code            | '-sleep(10)#
```

rails escapes the use of dangerous characters but still there is no reason to allow inserting such values in db in the first place.

LImit usage of special characters for such fields to allow only ascii  letters (or utf?), numbers and forward slash, apostrophe and hyphen (hope I did not leave out anything important).

**Comment by vohmar** [comment]
translation issues. English view:
![image](https://user-images.githubusercontent.com/7734747/73667391-dfb93e80-46ac-11ea-81a1-b4d1faa16458.png)

and it looks like we need some colon or smth to part the field name from the info message:
![image](https://user-images.githubusercontent.com/7734747/73667772-71c14700-46ad-11ea-982f-d4706c733783.png)

**Comment by vohmar** [comment]
fixed by #489
